Dockerfile

```diff
@@ -11,12 +11,20 @@ WORKDIR /dojo
 COPY . .
 RUN cargo build --release --config net.git-fetch-with-cli=true
 
+# Set an environment variable for the port
+ENV HEALTH_CHECK_PORT=5050
+
 FROM debian:buster-slim
 LABEL description="Dojo is a provable game engine and toolchain for building onchain games and autonomous worlds with Cairo" \
     authors="tarrence <tarrence@cartridge.gg>" \
     source="https://github.com/dojoengine/dojo" \
     documentation="https://book.dojoengine.org/"
 
+HEALTHCHECK --interval=10s --timeout=30s --start-period=10s --retries=10 \
```

Comment:
If interval is 10s, and timeout is 30s, how can you retry 10 times? (im a docker noob)

Reply:
Good catch! Actually we need to configure the start-period as well, we don't need 10s for the first healthcheck for katana